compiler/rustc_builtin_macros/src/format.rs

```diff
@@ -659,7 +690,11 @@ fn report_missing_placeholders(
     if !found_foreign && unused.len() == 1 {
         diag.span_label(fmt_span, "formatting specifier missing");
     }
-
+    if dup_exist {
+        diag.note("the formatting string captures that binding directly, it doesn't need to be included in the argument list");
+        let this_these = if duplicate_explicit_arg.len() == 1 { "this" } else { "these" };
+        diag.span_help(duplicate_explicit_arg, format!("Consider removing {}", this_these));
```

Comment:
We don't use "title case". I would also expand on this.

Ideally, we would provide a structured suggestion for the removal, but then you have to deal with creating spans out of thin air to cover the commas and whitespace, and it gets messy quick.